Automattic/pocket-casts-android
# Sync episodes sort order podcast setting

## Description

Somehow episodes order podcast setting slipped through the cracks and wasn't synced properly.

## Testing Instructions

1. Have two devices D1 and D2.
2. Make sure that both devices are synced before starting the test by refreshing apps in the profile.
3. D1: Go to a podcast.
4. D1: Tap on the `Overflow menu (three dots)`.
5. D1: Change `Sort episodes` setting.
6. D1: Sync the device in the profile.
7. D2: Sync the device in the profile.
8. D2: Go to the same podcast.
9. D2: Episodes should be sorted correctly.

## Checklist
 
#### I have tested any UI changes...
